omod/src/main/java/org/openmrs/web/servlet/LoginServlet.java

```diff
@@ -178,7 +182,18 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 			catch (ContextAuthenticationException e) {
 				// set the error message for the user telling them
 				// to try again
-				httpSession.setAttribute(WebConstants.OPENMRS_ERROR_ATTR, "auth.password.invalid");
+		
+				String maximumAttempts = Context.getAdministrationService().getGlobalProperty(GP_MAXIMUM_ALLOWED_LOGINS, "7");
+				Integer maximumAlowedAttempts = Integer.valueOf(maximumAttempts);
+	           
+					if (loginAttemptsByUserName <= maximumAlowedAttempts) {				
+						httpSession.setAttribute(WebConstants.OPENMRS_ERROR_ATTR, "auth.password.invalid");
+				
+					}
+				
+					if (loginAttemptsByUserName > maximumAlowedAttempts) {
```

Comment:
I think use of if else here will be much better because we are to check for only one of the condition and not both separately.